prisma/schema.prisma

```diff
@@ -115,16 +119,100 @@ model Area {
   question             String?
   assessors_guide      String?
   stageId              Int
-  stage                Stage      @relation(fields: [stageId], references: [id])
+  stage                Stage            @relation(fields: [stageId], references: [id])
   artefacts            Artefact[]
+  AssessmentArea       AssessmentArea[]
```

Comment:
Yes, the relationship you described is what I intended. 

One distinct area could have multiple assessments due to the nature of having multiple users being able to assess the same area. Assessment table is linked to a projectid so we are also able to extract the information you described from the query.

Other than that, I can't say this would be useful to extract the scores as well given that there is a table for that (if we keep it?) @severynenko